src/Schema/Events/Tap.ts

```diff
@@ -206,3 +206,32 @@ export interface TappedConsign {
   /** The text of the tapped button */
   subject: string
 }
+
+/**
+ * A user taps the promo space on the iOS home screen
+ *
+ * This schema describes events sent to Segment from [[tappedPromoSpace]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedPromoSpace",
+ *    context_module: "promoSpace",
+ *    context_screen_owner_type: "home",
+ *    destination_screen_owner_type: "collection",
+ *    destination_screen_owner_slug: artists-impacted-museum-shows,
+ *    subject: "Explore the collection",
+ *    utm_campaign: "art-keeps-going"
```

Comment:
`utm_campaign` is perhaps the only property we won't need, but I included it because I'd imagine the marketing organization is going to want to keep track of engagement per campaign, and this is an easy way to do it. 

I spoke with them earlier though, and they said they are not currently 100% sure how they will operationalize the promo space yet, so I could also see a world in which we add this later or make it optional. If anyone has thoughts, let me know.

Reply:
Would the `utm_campaign` be available in the subsequent screen's path? Maybe we don't need a property here and we can just use the screen path like we do for other channels?

Reply:
Good point. I've removed it! 👍